opentelekomcloud/services/as/resource_opentelekomcloud_as_policy_v2.go

```diff
@@ -22,6 +23,9 @@ func ResourceASPolicyV2() *schema.Resource {
 		UpdateContext: resourceASPolicyV2Update,
 		DeleteContext: resourceASPolicyV2Delete,
 
+		CustomizeDiff: customdiff.All(
```

Comment:
You can delete ``customdiff.All`` and leave only the function that do the logic for customizeDiff.
``All`` is used when schema have several custom rules